docs/detections/alerts-view-details.asciidoc

```diff
@@ -1,10 +1,10 @@
 [[view-alert-details]]
 == View detection alert details
 
-To inspect a detection alert, click the *View details* button from the Alerts table. The alert details flyout appears with several options to view alert data.
+To inspect a detection alert, click the *View details* button from the Alerts table. The alert details flyout appears with a way to share the alert and several options to view alert data.
```

Comment:
I'm not sure this is necessary to call out in the intro paragraph — it's a cool feature, but probably not the main reason users will be opening the alert details flyout. We explain the feature in detail almost immediately below, which seems sufficient.

Reply:
That's fair. The main purpose and value of the flyout is to shore more info on the alert. I can remove this bit.

```suggestion
To inspect a detection alert, click the *View details* button from the Alerts table. The alert details flyout appears with several options to view alert data.
```